sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveResolutionSuite.scala

```diff
@@ -81,9 +81,13 @@ class HiveResolutionSuite extends HiveComparisonTest {
       .toDF().registerTempTable("caseSensitivityTest")
 
     val query = sql("SELECT a, b, A, B, n.a, n.b, n.A, n.B FROM caseSensitivityTest")
-    assert(query.schema.fields.map(_.name) === Seq("a", "b", "A", "B", "a", "b", "A", "B"),
+    assert(query.schema.fields.map(_.name) === Seq("a", "b", "a", "b", "a", "b", "a", "b"),
       "The output schema did not preserve the case of the query.")
```

Comment:
It is no problem to normalize table name and column names. I meant, for the case user explicitly specifies the case in the query like this test, don't we need to preserve it in the schema of the returned df? e.g., Some databases might use case-sensitivity enabled configuration. If we connect with it through jdbc driver, may it cause problem?